ProjectLighthouse.Servers.GameServer/Controllers/Slots/SlotsController.cs

```diff
@@ -280,7 +280,7 @@ public async Task<IActionResult> TeamPickedSlots([FromQuery] int pageStart, [Fro
             .Take(Math.Min(pageSize, 30));
         string response = Enumerable.Aggregate(slots, string.Empty, (current, slot) => current + slot.Serialize(gameVersion));
         int start = pageStart + Math.Min(pageSize, ServerConfiguration.Instance.UserGeneratedContentLimits.EntitledSlots);
-        int total = await StatisticsHelper.TeamPickCount(this.database);
+        int total = await StatisticsHelper.TeamPickCountForGame(this.database, this.GetToken().GameVersion);
```

Comment:
```suggestion
        int total = await StatisticsHelper.TeamPickCountForGame(this.database, token);
```
Token already exists in the current context so there's no need to call GetToken again